modules/feed/html/rollup.js

```diff
@@ -244,6 +244,7 @@ exports.create = function (api) {
       var renderedMessage = api.message.html.render(item, {
         compact: compactFilter(item),
         includeForks: false, // this is a root message, so forks are already displayed as replies
+        renderAsCard: true,
```

Comment:
this is the only contentious bit IMO - it's the option to pass to all the items to let them know to be cards in the rolled up view ...`renderAsCard` is not perfect but it is descriptive ... maybe it could be `renderAsCardIfCan` ergg 
# :nauseated_face: